lib/cartopy/io/ogc_clients.py

```diff
@@ -574,7 +588,24 @@ def _wmts_images(self, wmts, layer, matrix_set_name, extent,
         # Find which tile matrix has the appropriate resolution.
         tile_matrix_set = wmts.tilematrixsets[matrix_set_name]
         tile_matrices = tile_matrix_set.tilematrix.values()
-        meters_per_unit = METERS_PER_UNIT[tile_matrix_set.crs]
+        if tile_matrix_set.crs in METERS_PER_UNIT:
+            meters_per_unit = METERS_PER_UNIT[tile_matrix_set.crs]
+        elif ':EPSG:' in tile_matrix_set.crs:
+            epsg_num = tile_matrix_set.crs.split(':')[-1]
+            tms_crs = ccrs.epsg(int(epsg_num))
+            with warnings.catch_warnings():
+                warnings.simplefilter("ignore")
```

Comment:
What is the warning we are ignoring? We should at least mention why this is here with a comment.

Reply:
Yes, this is a UserWarning from .to_dict() from proj because the dict does not give all details of each projection. But we only check for the units here.
I added a comment.